terraform/Makefile

```
@@ -1,11 +1,13 @@
 TARGET_OS_ARCH:=$(shell go env GOOS)_$(shell go env GOARCH)
+UNSUPPORTED_PROVIDERS:=$(shell echo $SKIP_TERRAFORM_PROVIDERS)
```

```suggestion
UNSUPPORTED_PROVIDERS:=$(SKIP_TERRAFORM_PROVIDERS)
```

no need for shell here